official/wide_deep/wide_deep_run_loop.py

```diff
@@ -48,6 +48,12 @@ def define_wide_deep_flags():
   flags.DEFINE_boolean(
       name="download_if_missing", default=True, help=flags_core.help_wrap(
           "Download data to data_dir if it is not already present."))
+  flags.DEFINE_integer(
+      name="inter_op_parallelism_threads", short_name="inter", default=0,
+      help="Number of threads to use for inter-op parallelism. If left as default value of 0, the system will pick an appropriate number.")
```

Comment:
Would be helpful to get a description of inter-op and intra-op somewhere.

Reply:
Yes, added more information and helpful links in README. Please take a look.